doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_architecture_pm.rst

```diff
@@ -250,3 +244,105 @@ Power management benchmark
 **************************
 
 To benchmark the power consumption in *Idle* state, see :ref:`multicore_idle_test`.
+
+Power management optimization example
+*************************************
+
+This example focuses on a HID device use case (for example, a mouse)
+In this use case the typical workflow is the following:
+
+  1. Sample input data from sensors.
+  2. Construct a HID report.
+  3. Transmit the report over the radio.
+  4. Sleep until the next period (usually, less than 1 ms).
+
+The power consumption targets for this example are the following:
+
+* Sleep currents (5 V supply):
+
+  * Application core (Suspend-to-RAM): < 10 µA
+  * Radio core (Suspend-to-Idle, cache disabled): < 10 µA
+
+* Active currents (estimates at 5 V):
+
+  * Radio core in WFI: ~0.25 mA
+  * App core in WFI: ~0.15 mA
+  * Core wake-up every 1 ms: +0.05 mA
+  * ADC sample every 1 ms: +0.10 mA
+  * SPI transaction every 1 ms: +0.20 mA
+  * IPC message every 1 ms: +0.15 mA
```

Comment:
To unify terminology (WFI is an instruction that suspends the CPU, which leads to the Idle power state).

```suggestion
  * Radio core in Idle: ~0.25 mA
  * App core in Idle: ~0.15 mA
  * Core wake-up every 1 ms: +0.05 mA
  * ADC sample every 1 ms: +0.10 mA
  * SPI transaction every 1 ms: +0.20 mA
  * IPC message every 1 ms: +0.15 mA
```